Corrected IP ranges in document.

wmrichard opened the pull request:
Corrected IP ranges in document. Tested the tutorial on Azure Stack integrated system.

azurecla commented:
Hi __@ppatierno__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

deneha commented:
@ppatierno Thank you for this contribution!  I'll have the author review.

@sethmanheim Would you please review this change and let me know if I should merge it?  Thank you.

sethmanheim commented:
Looks good, OK to merge.

wmrichard committed:
Corrected IP ranges in document.

Corrected IP ranges in document. Tested the tutorial on Azure Stack integrated system.
